eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -139,21 +157,42 @@ stages:
                           
                           Write-Host "git status"
                           git status
-                        displayName: Copy updated files into vcpkg fork for PR
-                        condition: ne(variables['SkipVcpkgUpdate'], 'true')
 
-                      # This template does not need to check the 
-                      # SkipVcpkgUpdate variable, it will stop if there are no
-                      # changes in the git repo.
+                          # Commit changes
+                          Write-Host "git add -A"
+                          git add -A 
+                          Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -m `"Initial vcpkg commit for ${{ artifact.VcpkgPortName }}`""
```

Comment:
Is there something in the bootstrap-vcpkg.bat that requires the changes to already be committed?